applications/content-creator-applications/domain/contentCreatorApplication.js

```diff
@@ -1,38 +1,35 @@
 module.exports = function buildMakeContentCreatorApplication({ Id }) {
 
-	return function makeContentCreatorApplication({
-		id = Id.makeId(),
-		firstName,
-		lastName,
-		email,
-		motivation,
-		approved = false,
-		createdAt = new Date(),
-		modifiedAt = new Date()
-	}) {
+    return function makeContentCreatorApplication({
+        id = Id.makeId(),
+        name,
```

Comment:
I think it should be first name and last name here as well

Reply:
yes, but for now we only use name